desktop-widgets/profilewidget.cpp

```diff
@@ -8,6 +8,45 @@
 
 #include <QToolBar>
 #include <QHBoxLayout>
```

Comment:
comment on the commit message of ** profile: implement "empty state" in desktop's ProfileWidget **
this is missing an empty line before the SOB